firecracker/rootfs/docker_runtime/create_rootfs_image.sh

```diff
@@ -32,6 +32,9 @@ mount -t fuse-ext2 -o rw+ "${IMAGE}" "${MOUNT_POINT}"
 ########################################
 # Copy rootfs into dist.
 ########################################
-tar --create --gzip --file="${DIST_FILE}" "${IMAGE}"
+(
+    cd /tmp
+    tar --create --gzip --file "${DIST_FILE}" --directory "/tmp" "rootfs.ext4"
```

Comment:
I think the `cd /tmp` is redundant with the `--directory "/tmp"` option on `tar`.

Reply:
believe it or not the `--directory` doesn't seem to do it itself

Reply:
Oh, try moving `--directory` further over to the left (before `--file`, at least):
> -C, --directory=DIR
              Change to DIR before performing any operations.  This option is order-sensitive, i.e. it affects all options that follow.